traffic_ops/traffic_ops_golang/dbhelpers/db_helpers_test.go

```diff
@@ -381,14 +381,14 @@ func TestGetCDNIDFromName(t *testing.T) {
 				mock.ExpectQuery("SELECT").WillReturnRows(rows)
 			}
 			mock.ExpectCommit()
-			_, exists, err := GetCDNIDFromName(db.MustBegin().Tx, "testCdn")
-			if testCase.storageError != nil && err == nil {
+			id, exists, err := GetCDNIDFromName(db.MustBegin().Tx, "testCdn")
+			if testCase.storageError != nil && err == nil && id != 0 {
 				t.Errorf("Storage error expected: received no storage error")
 			}
-			if testCase.storageError == nil && err != nil {
+			if testCase.storageError == nil && err != nil && id != 0 {
```

Comment:
In this case and the one above it, you can actually check if the `id` value is `1` or not. It should be 1 for the cases where `found` is true, because you specify it in this line:
`mock.ExpectQuery("SELECT").WillReturnRows(rows)`

Reply:
Good call. I've since made the expected ID value explicit.